makefile

```diff
@@ -1,29 +1,134 @@
 #!make
 
--include .env
+ENV_NAME ?= dev
 
-export $(shell sed 's/=.*//' .env)
+
+PROJECT_NAME = ets
```

Comment:
```suggestion
export PROJECT_NAME = ets
```

This in addition to renaming the usages in docker-compose should fix `run-local`

Reply:
TODO mentioned in makefile